user_journey/user_journey.model.lkml

```diff
@@ -6,6 +6,25 @@ include: "/user_journey/views/*.view.lkml"
 
 explore: funnel_analysis {
   view_label: " User-Day Funnels"
```

Comment:
```suggestion
  view_label: "User-Day Funnels"
```

Reply:
I added this space a dumb hack to have the view appear at the top of the views list, rather than the bottom:
<img width="314" alt="Screen Shot 2021-04-26 at 6 06 36 PM" src="https://user-images.githubusercontent.com/20819040/116156946-2a601580-a6ba-11eb-9ec9-4167ad150fff.png">

Reply:
Ah okay :D